src/index.html

```diff
@@ -15,8 +15,91 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        class="logo-image"
+        href="index.html"
+      >
+        <img
+          src="./images/logo.png"
+          alt="Логотип магазину MOYO"
```

Comment:
The 'alt' attribute value should be in English to maintain consistency with the language defined in the <html> tag (lang="en").